recipes/lima/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "lima" %}
```

Comment:
I believe a prohibitive concern here is that the name `lima` is used for [a different recipe in Bioconda](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/lima) that builds [a `lima` package from PacificBiosciences](https://github.com/PacificBiosciences/barcoding). Such a convergence is problematic for Bioconda because they prioritize the `conda-forge` channel, such that this package would mask theirs.

Can we consider shipping this as `lima-vm` instead?

Reply:
Yes, I actually meant to specifically ask if `lima-vm` should be used instead but forgot to put that in my comment.  I will change the recipe name as you have suggested.  Thanks!